src/run_throttlebot.py

```diff
@@ -65,6 +67,8 @@ def convert_percent_to_raw(mr, current_mr_allocation, weight_change=0):
         return  weighting_to_blkio(weight_change, current_mr_allocation)
     elif mr.resource == 'NET':
         return weighting_to_net_bandwidth(weight_change, current_mr_allocation)
+    elif mr.resource == 'MEMORY':
+        return weighting_to_memory(weight_change, current_mr_allocation, mr.instances[0])
```

Comment:
Memory allocations should be changed for *all* instances of the service.

Reply:
Yes, so memory is allocated like the other services (they are applied to all instances). However, when weighting the memory allocation value, I am making an assumption that the containers of the same service are uniform (aka the idle memory usage of one container is the same for all). That is why I only take one instance into consideration when weighting the value.

Reply:
I see. Still, please maintain the abstraction. You can choose to parse a single item of that deployment inside the weighting_to_memory class.